README.md

```diff
@@ -1,3 +1,38 @@
+![Path18 logo](logo.png)
 
-# Path18 
-Web App for exposing young people to a variety of career paths :mortar_board:
+##### Web App for exposing young people to a variety of career paths :mortar_board:
+
+### Problem Statement:
+* Unexposure to variety of jobs and career paths for young people
+* Uncertainty around career prospects
+
+### User Journey:
+#### As a user I can browse a variety of career paths and view detailed information about the ones that I am interested in.
+
+### User Story:
+##### As a user I can...
+* read welcome page and instructions on how to use the app
+* view card with job and decide if I like / dislike it
+* click on card to view more details
+* View list of jobs I like
+* click on job to view details on the job and career path
+* read details instructions for the app in the help page
+
+### Wireframe:
+
+### Architecture:
+
+### Technologies:
+
+### Team Guidelines:
+
+* CamelCase for JavaScript id's
+* kebab-case for CSS Classes
+* comments for code clarification
+* ES6 on backend
+* all team member understand code before approval
+
+### Stretch Goals:
+* user authentication
+* use of logic for browsing job options
+* expand data to include colleges
```

Comment:
could add the ability to delete a job once you have been matched with it

Reply:
I didn't add it yet, but we can definitely add it later :wink: